internal/token/token_test.go

```
@@ -0,0 +1,365 @@
+		testCA = generateTestCA()
+		generateLeafCert(signingCertFile, signingKeyFile, "fulfillment-token-signer", testCA)
+		generateLeafCert(encryptionCertFile, encryptionKeyFile, "fulfillment-token-encryption", testCA)
+
+		issuer = "https://fulfillment.test.example.com"
+		audience = []string{"test-audience"}
+		sealer, err = NewSealer(logger, signingCertFile, signingKeyFile, encryptionCertFile, issuer, audience)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		os.RemoveAll(tmpDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)